chapters/interface.tex

```diff
@@ -21,51 +21,50 @@ \chapter{Interface or Type Relationships}\doublelabel{interface-or-type-relation
-that is needed to decide whether A can be used instead of B
-{[}\emph{E.g. a declaration ``}\lstinline!Real x!\emph{'' is part of the type, also
-called interface, but ``}\lstinline!import A!\emph{'' is not}{]}.
+that is needed to decide whether \lstinline!A! can be used instead of \lstinline!B!
+{[}\emph{E.g. a declaration} \lstinline!Real x! \emph{is part of the type, also
+called interface, but} \lstinline!import A! \emph{is not}{]}.
 \\ \hline
 \begin{tabular}{@{}p{4cm}@{}}
 class type or\\
 inheritance interface
 \end{tabular}
 &
 The ``essential'' part of the public \emph{and protected} declaration
-sections of a class that is needed to decide whether A can be used
-instead of B. The class type , also called inheritance interface, is
+sections of a class that is needed to decide whether \lstinline!A! can be used
+instead of \lstinline!B!. The class type, also called inheritance interface, is
 needed when inheritance takes place, since then the protected
 declarations have to be taken into account.\\ \hline
 \begin{tabular}{@{}p{4cm}@{}}
 subtype or\\
 compatible interface
 \end{tabular} &
-A is a subtype of B, or equivalently, the interface of A is compatible
-to the interface of B, if the ``essential'' part of the public
-declaration sections of B is also available in A {[}\emph{E.g., if B has
-a declaration ``}\lstinline!Real x!\emph{'', this declaration must also be present
-in A. If A has a declaration ``Real y'', this declaration must not be
-present in B}{]}.\\ \hline
+\lstinline!A! is a subtype of \lstinline!B!, or equivalently, the interface of \lstinline!A! is compatible
+to the interface of \lstinline!B!, if the ``essential'' part of the public
+declaration sections of \lstinline!B! is also available in \lstinline!A! {[}\emph{E.g., if \lstinline!B! has
+a declaration} \lstinline!Real x!\emph{, this declaration must also be present
+in \lstinline!A!. If \lstinline!A! has a declaration \lstinline!Real y!, this declaration must not be
+present in \lstinline!B!}{]}.\\ \hline
 \begin{tabular}{@{}p{4cm}@{}}
-restricted subtype or
-plug compatible interface
+restricted subtype or plug compatible interface
 \end{tabular} &
-A is a restricted subtype of B, or equivalently, the interface of A is
-plug compatible to the interface of B, if A is a subtype of B and if
-connector components in A that are not in B, are default connectable.
+\lstinline!A! is a restricted subtype of \lstinline!B!, or equivalently, the interface of \lstinline!A! is
+plug compatible to the interface of \lstinline!B!, if \lstinline!A! is a subtype of \lstinline!B! and if
+connector components in \lstinline!A! that are not in \lstinline!B!, are default connectable.
 {[}\emph{E.g. it is not allowed that these connectors have variables
-with the ``input'' prefix, because then they must be connected.}{]} A
-model or block A cannot be used instead of B, if the particular
+with the \lstinline!input! prefix, because then they must be connected.}{]} \lstinline!A!
```

Comment:
This A should not be code, it's the word "a" at the start of a sentence.

Reply:
True.  Fixed.